api/src/main/resources/liquibase.xml

```diff
@@ -307,82 +307,20 @@
         </insert>
     </changeSet>
 
-	<changeSet author="rmakoba" id="ugandaemr-04052020-iss-9">
-		<preConditions onFail="MARK_RAN">
```

Comment:
@Mwanje This should be MARK_RAN since if the data exists then there is no need to run this changeset